File: ServerSpace/src/main/java/Models/Servicio/ServicioMainModel.java
package Models.Servicio;


import java.sql.Timestamp;

import com.fasterxml.jackson.annotation.JsonProperty;

import EntityLayer.ServicioEntity;

public class ServicioMainModel {


    public ServicioMainModel() {
        this.servicioId = 0;
        this.correlativo = "";
        this.nombre = "";
        this.descripcion = "";
        this.tipoServicio= "";
        this.fechaRegistro = null;
        this.codUsuario = "";
        this.estadoRegistro = false;
    }

    public ServicioMainModel(ServicioEntity Ent) {
        this.servicioId = Ent.getServicioId();
        this.correlativo = Ent.getCorrelativo();
        this.nombre = Ent.getNombre();
        this.descripcion = Ent.getDescripcion();
        this.tipoServicio= Ent.getTipoServicio();
        this.fechaRegistro = Ent.getFechaRegistro();
        this.codUsuario = Ent.getCodUsuario();
        this.estadoRegistro = Ent.getEstadoRegistro();
    }


    @JsonProperty("ServicioId")
    private int servicioId = 0;

    public int getServicioId() {
        return servicioId;
    }

    public void setServicioId(int servicioId) {
        this.servicioId = servicioId;
    }

    @JsonProperty("Correlativo")
    private String correlativo = "";

    public String getCorrelativo() {
        return correlativo;
    }

    public void setCorrelativo(String correlativo) {
        this.correlativo = correlativo;
    }

    @JsonProperty("Nombre")
    private String nombre = "";

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    @JsonProperty("Descripcion")
    private String descripcion = "";

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    @JsonProperty("TipoServicio")
    private String tipoServicio = "";

    public String getTipoServicio() {
        return tipoServicio;
    }

    public void setTipoServicio(String tipoServicio) {
        this.tipoServicio = tipoServicio;
    }
    @JsonProperty("FechaRegistro")
    private Timestamp fechaRegistro = null;

    public Timestamp getFechaRegistro() {
        return fechaRegistro;
    }

    public void setFechaRegistro(Timestamp fechaRegistro) {
        this.fechaRegistro = fechaRegistro;
    }

    @JsonProperty("CodUsuario")
    private String codUsuario = "";

    public String getCodUsuario() {
        return codUsuario;
    }

    public void setCodUsuario(String codUsuario) {
        this.codUsuario = codUsuario;
    }

    @JsonProperty("EstadoRegistro")
    private boolean estadoRegistro = false;

    public boolean getEstadoRegistro() {
        return estadoRegistro;
    }

    public void setEstadoRegistro(boolean estadoRegistro) {
        this.estadoRegistro = estadoRegistro;
    }



}

File: SharedApi/Space/Space.Server/Model/Servicio/ServicioMainModel.cs
using Space.EntityLayer;
using System.Text.Json.Serialization;

namespace Space.Server
{
    public class ServicioMainModel
    {

        public ServicioMainModel(ServicioEntity Item)
        {
            this.ServicioId = Item.ServicioId;
            this.Correlativo = Item.Correlativo;
            this.Nombre = Item.Nombre;
            this.Descripcion = Item.Descripcion;
            this.TipoServicio = Item.TipoServicio;
            this.FechaRegistro = Item.FechaRegistro;
            this.CodUsuario = Item.CodUsuario;
            this.EstadoRegistro = Item.EstadoRegistro;
        }

        [JsonPropertyName("ServicioId")] public Int32 ServicioId { get; set; }
        [JsonPropertyName("Correlativo")] public String Correlativo { get; set; }
        [JsonPropertyName("Nombre")] public String Nombre { get; set; }
        [JsonPropertyName("Descripcion")] public String Descripcion { get; set; }
        [JsonPropertyName("TipoServicio")] public String TipoServicio { get; set; }
        [JsonPropertyName("FechaRegistro")] public DateTime FechaRegistro { get; set; }
        [JsonPropertyName("CodUsuario")] public String CodUsuario { get; set; }
        [JsonPropertyName("EstadoRegistro")] public Boolean EstadoRegistro { get; set; }
    }
}
